jetty-server/src/main/java/org/eclipse/jetty/server/HttpConfiguration.java

```diff
@@ -57,7 +57,7 @@
     private int _headerCacheSize=512;
     private int _securePort;
     private long _idleTimeout=-1;
-    private long _blockingTimeout=-1;
+    private long _blockingTimeout= 10*60*1000;
```

Comment:
What is sane and "production ready" for one person is a "heinous crime against performance" for another.

BlockingTimeout is configurable from the `${jetty.base}` configuration directory (if using jetty-home or jetty-distribution), and embedded-jetty already.

Reply:
Who on earth wants infinite blocking of its server? :)

I know I biased... we launch Jetty from our Java service.